ports/unix/main.c

```diff
@@ -142,23 +141,22 @@ STATIC int execute_from_lexer(int source_kind, const void *source, mp_parse_inpu
 
         if (!compile_only) {
             // execute it
-            mp_call_function_0(module_fun);
+            mp_obj_t ret = mp_call_function_0(module_fun);
```

Comment:
`module_fun` can be `MP_OBJ_NULL` at this point since `mp_compile` can return that so that should be checked for before reaching this line.

Reply:
Thanks, now fixed.